react/src/components/DefaultProviders.tsx

```diff
@@ -118,7 +119,7 @@ export const useCurrentLanguage = () => {
     return () => window.removeEventListener('langChanged', handler);
   }, [i18n]);
 
-  return [lang] as const;
+  return { lang, setLang } as const;
```

Comment:
Is there a reason to change this part from an array to an object?
`[lang, setLang]` vs. `{lang, setLang}`

Reply:
no... I'll revert that.